diff --git a/node_modules/react-native-live-audio-stream/RNLiveAudioStream.podspec b/node_modules/react-native-live-audio-stream/RNLiveAudioStream.podspec
index f7307f5..fa651fa 100644
--- a/node_modules/react-native-live-audio-stream/RNLiveAudioStream.podspec
+++ b/node_modules/react-native-live-audio-stream/RNLiveAudioStream.podspec
@@ -13,10 +13,10 @@ Pod::Spec.new do |s|
   s.source         = { :git => 'https://github.com/xiqi/react-native-live-audio-stream.git' }
 
   s.requires_arc   = true
-  s.ios.deployment_target = '8.0'
+  s.ios.deployment_target = '13.4'
 
   s.preserve_paths = 'README.md', 'package.json', 'index.js'
   s.source_files   = 'iOS/*.{h,m}'
 
-  s.dependency 'React'
+  s.dependency 'React-Core'
 end
